python/paddle/fluid/layers/control_flow.py

```diff
@@ -2128,62 +2329,88 @@ def memory(self,
                need_reorder=False,
                dtype='float32'):
         """
-        Create a memory variable for dynamic rnn.
+        Create a memory Variable for DynamicRNN to deliver data cross time steps.
+        It can be initialized by a existing Tensor or to a constant Tensor of given
+        dtype and shape.
 
-        If the :code:`init` is not None, :code:`memory` will be initialized by
-        this variable. The :code:`need_reorder` is used to reorder the memory as
-        the input variable. It should be set to true when the initialized memory
-        depends on the input sample.
+        Args:
+            init (Variable, optional): LoDTensor used to initialize the memory.
+                If init is not None, it should hold the same number of sequences
+                as RNN's input (the input LoDTensor set by :code:`step_input()` )
+                and the memory will be initialized to it. If init's LoD is null,
```

Comment:
Same comment as above:

Should we say "None" instead of "null" in Python?

Reply:
Done.